### ISSUE TITLE
Ability to hide own boosts from timeline

### ISSUE THREAD

**Issue by debbiegoldsmith** [mastodon/mastodon#20093]
### Pitch

I know I boosted it. I don't need to see it again. It would be fine to show it in "posts" in my profile, but I don't want to see it in my timeline.

It feels more like a pref setting than something you'd pick from the menu on your own profile, but that would be OK. (Note that you can hide boosts from other people via their profile, but not your own!)

### Motivation

When trying to catch up on my timeline I find myself reading something when I realize it's one of my own boosts and I already read it.

**Comment by MikeTaylor** [comment]
+1

**Comment by bnlawrence** [comment]
+1

**Comment by CEbbinghaus** [comment]
I would propose extending this to being able to hide ALL my activity from the various timelines.

**Comment by LeslieFH** [comment]
I came here to suggest this option, I've seen a post I boosted three times already in my timeline. I boosted it, I know it exists, I don't need it on my timeline anymore.

**Comment by CEbbinghaus** [comment]
> I've seen a post I boosted three times already in my timeline.

There is a separate issue tracking Seeing the same content more than once. 

I have been trying to write this functionality myself but I am still stuck on getting the dev environment to work. If I manage to implement this functionality I will update this issue

**Comment by MikeTaylor** [comment]
@CEbbinghaus:

> There is a separate issue tracking Seeing the same content more than once.

Where is it? I've not been able to find this by searching for the title you suggested.

**Comment by 1Syler** [comment]
I just came here from googling if this can be done. As others have said, there doesn't seem to be any logical reason for showing yourself your own boosts.

**Comment by CEbbinghaus** [comment]
> Where is it? I've not been able to find this by searching for the title you suggested.

#8944 Seems to contain a similar problem although I haven't been able to find a Issue that mentions this exact problem. I know there is a decent amount of support for this feature so I might create a new issue specifically for hiding duplicate posts (although determining which one should be kept might be hard)

**Comment by lookitmychicken** [comment]
yes I really don't want to see boosts when I look at my profile, either. or have the ability to see them in a separate tab, like replies go in a separate tab. I want to be able to quickly find what I've posted recently, without having to scroll past all the stuff I've boosted.

**Comment by beren12** [comment]
Any updates on this? I used to be able to temp mute myself on Tweetbot which was super helpful in cleaning up my feed on twitter. Maybe similar here?

**Comment by nehemiagurl** [comment]
there's some peculiarity to the fact you can unfollow, hide boosts and mute all accounts on the Fediverse, except yourself.
